megamek/i18n/megamek/common/options/messages.properties

```diff
@@ -555,6 +555,8 @@ PilotOptionsInfo.option.tm_frogman.displayableName=Terrain Master [Frogman] (Cam
 PilotOptionsInfo.option.tm_frogman.description=Movement in depth 2 or greater water costs 1 MP less.\nAlso grants -1 to all PSR checks (including physicals) as well as -2 to Crush Depth checks.
 PilotOptionsInfo.option.tm_mountaineer.displayableName=Terrain Master [Mountaineer] (CamOps)
 PilotOptionsInfo.option.tm_mountaineer.description=Movement in rubble or rough cost 1 less MP and elevation changes also cost 1 MP less.\nThe mountaineer also recieves a -1 to PSR checks for the given terrain.
+PilotOptionsInfo.option.tm_nightwalker.displayableName=Terrain Master [Nightwalker] (CamOps)
+PilotOptionsInfo.option.tm_nightwalker.description=Nightwalker ignores all darkness move penalties when using Walk/Cruise movement but does not affect gunnery modifiers for lighting conditions.
```

Comment:
```suggestion
PilotOptionsInfo.option.tm_nightwalker.description=Nightwalker ignores all darkness move penalties when using Walk/ Cruise movement but does not affect gunnery modifiers for lighting conditions.
```

Reply:
Written with the same spacing usage in the rules. I can change it if you want, just following how it is written elsewhere.

Reply:
It should never be written without at least one space, and both sides is preferrable. It's an accessibility issue, as without the spacing the words merge together.

Reply:
Ok then, understandable